nist_800_171_rev_2/cm.sp

```diff
@@ -0,0 +1,94 @@
+benchmark "nist_800_171_rev_2_3_4" {
+  title       = "3.4 Configuration Management"
+  description = "CM controls are specific to an organization's configuration management policies. This includes a baseline configuration to operate as the basis for future builds or changes to information systems. Additionally, this includes information system component inventories and a security impact analysis control."
+  children = [
+    benchmark.nist_800_171_rev_2_3_4_1,
+    benchmark.nist_800_171_rev_2_3_4_2,
+    benchmark.nist_800_171_rev_2_3_4_6,
+    benchmark.nist_800_171_rev_2_3_4_7,
+    benchmark.nist_800_171_rev_2_3_4_9
+  ]
+
+  tags = local.nist_800_171_rev_2_common_tags
+}
+
+benchmark "nist_800_171_rev_2_3_4_1" {
+  title       = "3.4.1 Establish and maintain baseline configurations and inventories of organizational systems (including hardware, software, firmware, and documentation) throughout the respective system development life cycles"
```

Comment:
```suggestion
  title       = "3.4.1 Establish and maintain baseline configurations and inventories of organizational systems (including hardware, software, firmware, and documentation) throughout the respective system development lifecycles"
```

Reply:
@misraved this is SDLC -- so mostly written as `Systems development life cycle`